README.md

```diff
@@ -1 +1,40 @@
-# Hello-Microverse-Reviewable
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> A basic Hello Microverse Project with HTML and CSS. It has Linters Implemented and ignored redundant files using .gitignore
+
+![screenshot]()
+
+
+## Built With
+
+- VSCODE
+- Github
+- Git
+
+## Getting Started
+
+
+## Authors
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
```

Comment:
[OPTIONAL]: Add a hat-tip on the acknowledgments section to show your gratitude to someone you appreciate ~ or, remove this from your README so it doesn't seem like you copied from a template! 😉